docs/reference/analysis/tokenfilters/asciifolding-tokenfilter.asciidoc

```diff
@@ -23,9 +96,20 @@ PUT /asciifold_example
 }
 --------------------------------------------------
 
-Accepts `preserve_original` setting which defaults to false but if true
-will keep the original token as well as emit the folded token.  For
-example:
+[[analysis-asciifolding-tokenfilter-configure-parms]]
+==== Configurable parameters
+
+`preserve_original`::
+(Optional, boolean)
+If `true`, keep the original tokens and emit folded tokens.
+Defaults to `false`.
+
+[[analysis-asciifolding-tokenfilter-customize]]
+==== Customize
+
+To customize the ASCII folding token filter, duplicate it to create the basis
+for a new custom token filter. You can modify the filter using its configurable
+parameters.
 
```

Comment:
Here again, I'd introduce the example by calling out what it does. Something like:

For example, the following request creates a custom `asciifolding` filter with `preserve_original` set to `true`: